src/JitInterface/src/CorInfoImpl.cs

```diff
@@ -2745,8 +2745,20 @@ private void getLocationOfThisType(out CORINFO_LOOKUP_KIND result, CORINFO_METHO
 
         private void* getPInvokeUnmanagedTarget(CORINFO_METHOD_STRUCT_* method, ref void* ppIndirection)
         { throw new NotImplementedException("getPInvokeUnmanagedTarget"); }
+
         private void* getAddressOfPInvokeFixup(CORINFO_METHOD_STRUCT_* method, ref void* ppIndirection)
-        { throw new NotImplementedException("getAddressOfPInvokeFixup"); }
+        {
+            MethodDesc md = HandleToObject(method);
+
+            string externName = md.GetPInvokeMethodMetadata().Name ?? md.Name;
```

Comment:
It maybe better to just call getAddressOfPInvokeTarget from here instead of duplicating the code here.